dashboard/test/controllers/congrats_controller_test.rb

```diff
@@ -30,7 +30,7 @@ class CongratsControllerTest < ActionController::TestCase
 
     Unit.stubs(:should_cache?).returns(true)
 
-    assert_cached_queries(0) do
+    assert_cached_queries(1) do
```

Comment:
I didn't investigate why this number is going up, but it is on a codepath that is less performance-critical for HOC, so I think it is a worthwhile tradeoff.